core/src/main/java/com/sequenceiq/cloudbreak/controller/validation/template/InstanceTemplateValidator.java

```diff
@@ -1,17 +1,24 @@
 package com.sequenceiq.cloudbreak.controller.validation.template;
 
 import java.util.Objects;
+import java.util.stream.Collectors;
 
+import org.apache.commons.collections.CollectionUtils;
 import org.springframework.stereotype.Component;
 
 import com.sequenceiq.cloudbreak.domain.Template;
+import com.sequenceiq.cloudbreak.domain.VolumeTemplate;
 import com.sequenceiq.cloudbreak.validation.ValidationResult;
 import com.sequenceiq.cloudbreak.validation.ValidationResult.ValidationResultBuilder;
 import com.sequenceiq.cloudbreak.validation.Validator;
+import com.sequenceiq.cloudbreak.validation.VolumeParameterConstants;
 
 @Component
 public class InstanceTemplateValidator implements Validator<Template> {
 
+    static final String INVALID_NUMBER_OF_VOLUMES_MESSAGE_PATTERN = "Maximum '%s' volumes could be configured for an instance. "
+            + "Please modify the instance template with name: '%s', it has '%s' volumes configured!";
```

Comment:
The first and third `%s` should be replaced with `%d` as those are integers (`long` & `Integer`).